tools/pack_model_artifacts.py

```diff
@@ -0,0 +1,83 @@
+"""
```

Comment:
Can we move this file under python/graphstorm/sagemaker? So it can be installed with pip.
And also rename the file to pack_sm_model_artifacts.py or pack_sagemaker_model_artifacts.py.

Reply:
This script will not be used in the realtime inference pipeline. Instead, it provides users some flexibility to pack model artifacts on their own.
The same wrapping function will be used in launch endpoint script that is part of the sagemaker core code.

Reply:
Is this targeted only at SageMaker then? If that's the case we can put it under the `sagemaker` directory, under an `endpoint` or `real-time-inference` path? Also see other comment about asking for entry points explicitly.

Reply:
reorg folder locations.